### ISSUE TITLE
Cannot compile using zig master build

### ISSUE THREAD

**Issue by artix-boi** [cryptocode/bio#2]
After some minor fixes like fixing unused function errors,etc. Bio would not compile and it errors out with
`Semantic Analysis [531/911] Unreachable at /home/vsts/work/1/s/src/stage1/analyze.cpp:892 in get_slice_type. This is a bug in the Zig compiler.thread 1520 panic: 
Unable to dump stack trace: debug info stripped
bio...The following command terminated unexpectedly:
/workspace/bio/zig/zig build-exe /workspace/bio/src/main.zig -lc -cflags -std=c99 -- /workspace/bio/deps/linenoise.c --cache-dir /workspace/bio/zig-cache --global-cache-dir /home/gitpod/.cache/zig --name bio -I /workspace/bio/deps --enable-cache 
error: the following build command failed with exit code 6:
/workspace/bio/zig-cache/o/cd6f1c31aba6ac98cbd2100f0f46b5d3/build /workspace/bio/zig/zig /workspace/bio /workspace/bio/zig-cache /home/gitpod/.cache/zig`

**Comment by cryptocode** [comment]
Yep, Zig is currently a fast moving target. I'll try to get the project updated and compiling again soon, unless someone is able to do a PR first 😉 

Thanks for the report!

**Comment by artix-boi** [comment]
thank you thank you!
